cmd/create/accountroles/cmd.go

```diff
@@ -0,0 +1,472 @@
+		if err != nil {
+			return err
+		}
+
+		reporter.Debugf("Saving '%s' to the current directory", filename)
+		err = saveDocument(policy, filename)
+		if err != nil {
+			return err
+		}
+	}
+
+	for credrequest := range aws.CredentialRequests {
+		filename := fmt.Sprintf("openshift_%s_policy.json", credrequest)
+		path := fmt.Sprintf("templates/policies/%s/%s", version, filename)
+
+		policy, err := aws.ReadPolicyDocument(path)
+		if err != nil {
+			return err
+		}
+
+		reporter.Debugf("Saving '%s' to the current directory", filename)
+		err = saveDocument(policy, filename)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func saveDocument(doc []byte, filename string) error {
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	_, err = file.Write(doc)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func buildCommands(prefix string, version string) string {
+	commands := []string{}
+
+	for file, role := range aws.AccountRoles {
+		name := getRoleName(prefix, role)
+		iamTags := fmt.Sprintf(
+			"Key=%s,Value=%s Key=%s,Value=%s Key=%s,Value=%s",
+			tags.OpenShiftVersion, version,
+			tags.RolePrefix, prefix,
+			tags.RoleType, file,
+		)
+		createRole := fmt.Sprintf("aws iam create-role \\\n"+
+			"\t--role-name %s \\\n"+
+			"\t--assume-role-policy-document file://sts_%s_trust_policy.json \\\n"+
+			"\t--tags %s",
+			name, file, iamTags)
+		putRolePolicy := fmt.Sprintf("aws iam put-role-policy \\\n"+
+			"\t--role-name %s \\\n"+
+			"\t--policy-name %s-Policy \\\n"+
+			"\t--policy-document file://sts_%s_permission_policy.json",
+			name, name, file)
+		commands = append(commands, createRole, putRolePolicy)
+	}
+
+	for credrequest, operator := range aws.CredentialRequests {
+		name := getPolicyName(prefix, operator.Namespace, operator.Name)
+		iamTags := fmt.Sprintf(
+			"Key=%s,Value=%s Key=%s,Value=%s Key=%s,Value=%s Key=%s,Value=%s",
+			tags.OpenShiftVersion, version,
+			tags.RolePrefix, prefix,
+			"operator_namespace", operator.Namespace,
+			"operator_name", operator.Name,
+		)
+		createPolicy := fmt.Sprintf("aws iam create-policy \\\n"+
+			"\t--policy-name %s \\\n"+
+			"\t--policy-document file://openshift_%s_policy.json \\\n"+
+			"\t--tags %s",
+			name, credrequest, iamTags)
+		commands = append(commands, createPolicy)
+	}
+
+	return strings.Join(commands, "\n\n")
+}
+
+func createRoles(reporter *rprtr.Object, awsClient aws.Client, prefix string, version string, env string) error {
+	for file, role := range aws.AccountRoles {
+		name := getRoleName(prefix, role)
+
+		if !confirm.Confirm("create the '%s' role", name) {
+			continue
+		}
+
+		filename := fmt.Sprintf("sts_%s_trust_policy.json", file)
+		path := fmt.Sprintf("templates/policies/%s", filename)
+
+		policy, err := aws.ReadPolicyDocument(path, map[string]string{
+			"aws_account_id": aws.JumpAccounts[env],
+		})
+		if err != nil {
+			return err
+		}
+
+		reporter.Infof("Creating role '%s'", name)
```

Comment:
@vkareh since `rosa create cluster` takes flags with role ARN's should we output them here so a user doesn't have to go look them up?

Reply:
that's a good idea, yes